File: February 23/10 Feb/Balloon Everywhere gfg.cpp
    int maxInstance(string s){
        vector<int> count(5,0);
        int n =s.size();
        for(int i=0;i<n;i++)
        {
            if(s[i]=='b')
            count[0]++;
            else if(s[i]=='a')
            count[1]++;
            else if(s[i]=='l')
            count[2]++;
            else if(s[i]=='o')
            count[3]++;
            else if(s[i]=='n')
            count[4]++;
        }
        return min({count[0],count[1],count[2]/2,count[3]/2,count[4]});
    }

File: February 23/10 Feb/Balloon Everywhere gfg.py
def maxInstance(self, s : str) -> int:
        # code here
        count_b=0
        count_a=0
        count_l=0
        count_o=0
        count_n=0
        
        for ch in s:
            if(ch == 'b'):
                count_b+=1;
            elif(ch=='a'):
                count_a+=1;
            elif(ch == 'l'):
                count_l+=1;
            elif(ch == 'o'):
                count_o+=1;
            elif(ch == 'n'):
                count_n+=1;
        
        return int(min({count_b,count_a,count_l/2,count_o/2,count_n}))
